torch/csrc/autograd/python_function.cpp

```diff
@@ -845,6 +864,7 @@ PyObject* getRequiresGrad(PyObject* obj, void* _unused) {
 static struct PyGetSetDef THPFunction_properties[] = {
   {"saved_tensors", (getter)THPFunction_saved_tensors, nullptr, nullptr, nullptr},
   {"saved_variables", (getter)THPFunction_saved_variables, nullptr, nullptr, nullptr},
+  {"raw_saved_tensors", (getter)THPFunction_raw_saved_tensors, nullptr, nullptr, nullptr},
```

Comment:
Maybe add a small test to make sure we can access this from `grad_fn.raw_saved_tensor` and test what happens if there aren't any saved etc.

Reply:
Why is this public API? Shouldn't it be `_raw_saved_tensors` ?

Reply:
I used `raw_saved_tensors` for consistency with the implicit rule "prefix `saved` with `raw_` and you get the raw version". Would you like me to change it?